modules/diskq/diskq-grammar.ym

```diff
@@ -96,6 +101,20 @@ dest_diskq_option
         | KW_DISK_BUF_SIZE '(' nonnegative_integer64 ')' { disk_queue_options_disk_buf_size_set(last_options, $3); }
         | KW_QOUT_SIZE '(' nonnegative_integer ')'       { disk_queue_options_qout_size_set(last_options, $3); }
         | KW_DIR '(' string ')'                          { disk_queue_options_set_dir(last_options, $3); free($3); }
+        | KW_TRUNCATE_SIZE_RATIO '(' float_between_0_and_1 ')' { disk_queue_options_set_truncate_size_ratio(last_options, $3); }
+        ;
+
+diskq_global_options
+        : diskq_global_option diskq_global_options
+        |
+        ;
+
+diskq_global_option
+        : KW_TRUNCATE_SIZE_RATIO '(' float_between_0_and_1 ')' { disk_queue_config_set_truncate_size_ratio(configuration, $3); }
+        ;
+
+float_between_0_and_1
+        : nonnegative_float { CHECK_ERROR(($1 <= 1), @1, "It cannot be bigger, than 1"); }
```

Comment:
Shouldn't we set the result of the rule here? `$$ = `

Reply:
From https://www.gnu.org/software/bison/manual/html_node/Actions.html:

If you don’t specify an action for a rule, Bison supplies a default: $$ = $1. Thus, the value of the first symbol in the rule becomes the value of the whole rule. Of course, the default action is valid only if the two data types match. There is no meaningful default action for an empty rule; every empty rule must have an explicit action unless the rule’s value does not matter.

So I think it is implicitly set.

Reply:
Thanks for the clarification.